hardware_interface/include/hardware_interface/utils/ros2_control_utils.hpp

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020 ROS2-Control Development Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/* This header must be included by all rclcpp headers which declare symbols
+ * which are defined in the rclcpp library. When not building the rclcpp
+ * library, i.e. when using the headers in other package's code, the contents
+ * of this header change the visibility of certain symbols which the rclcpp
+ * library cannot have, but the consuming code must have inorder to link.
+ */
+
+#ifndef HARDWARE_INTERFACE__UTILS__ROS2_CONTROL_UTILS_HPP_
+#define HARDWARE_INTERFACE__UTILS__ROS2_CONTROL_UTILS_HPP_
+
+#include <map>
+#include <memory>
+#include <string>
+
+#include "pluginlib/class_loader.hpp"
+#include "hardware_interface/visibility_control.h"
+
+// TODO(destogl): Create util library instead of inline functions?
+
+namespace ros2_control_utils
+{
+// Classes
+template<typename T>
+class ROS2ControlLoaderPluginlib
```

Comment:
There was some use before but currently seams that only wraps in a very simple way the pluginlib. I can remove this.

P.S. 
> Everytime there's the word utils/helpers/tools it just means that the author couldn't categorize that piece of code well.

I just have to smile, because is so true :dagger:

Reply:
I would second Bence's concern about removing this class as for now.